### DISCUSSION TITLE
1.4.0

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#266] [Обновления]
## Список изменений
- Добавлена поддержка субтитров (От @MrSoczekXD в [#247](https://github.com/ilyhalight/voice-over-translation/pull/247))
- Добавлена подсветка слов в субтитрах (От @MrSoczekXD в [#247](https://github.com/ilyhalight/voice-over-translation/pull/247))
- Добавлена установка максимальной длинны субтитров (От @MrSoczekXD в [#247](https://github.com/ilyhalight/voice-over-translation/pull/247))
- Фикс громкости на ютубе (От @SashaXser в [#262](https://github.com/ilyhalight/voice-over-translation/pull/262))
- Добавлено сохранение языка, на который будет переводиться видео
- Были добавлены новые языки для перевода меню:
  - Бенгальский
  - Малайский
  - Яванский
  - Урду
  - Персидский
  - Индонезийский
- Добавлен ESLint для облегчения нахождения и исправления ошибок во время разработки
- Домен воркера был изменен на vot-new.toil-dump.workers.dev, дабы не нарушать работу у тех, кто пользуется старыми версиями расширения
- Некоторые улучшения связанные с структурой и запросами (От @MrSoczekXD в [#247](https://github.com/ilyhalight/voice-over-translation/pull/247))

## What's Changed (autogenerated)
* Update README.md by @miketype in https://github.com/ilyhalight/voice-over-translation/pull/260
* Фикс громкости by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/262
* Add subtitles support by @MrSoczekXD in https://github.com/ilyhalight/voice-over-translation/pull/247

## New Contributors
* @miketype made their first contribution in https://github.com/ilyhalight/voice-over-translation/pull/260
* @MrSoczekXD made their first contribution in https://github.com/ilyhalight/voice-over-translation/pull/247

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.3.6...1.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.4.0'>1.4.0</a>.</em>

**Comment by NullVerdict** [comment]
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/a34d4d0b-21c2-4210-b06d-5837aa1a3710)

**Reply by ilyhalight** [reply]
Исправил
